docs/app/pages/components/components-sections/select/select-custom/snippets/app.html

```diff
@@ -0,0 +1,76 @@
+<div class="row">
+    <div class="col-md-5">
+
+        <ux-select-custom
+            [(selected)]="selected"
+            (filterChange)="setFilter($event)"
+            [allowNull]="allowNull"
+            [busy]="showBusyIndicator"
+            [maxHeight]="componentMaxHeight"
+            [placeholder]="placeholder">
+
+            <ng-template #buttonContent>
+                <b>Selection:</b> {{ selected ? selected.name : '(none)' }}
+            </ng-template>
+
+            <ng-template #dropdownContent>
+
+                <div class="radio-button-container"
+                     uxTabbableList>
+                    <ux-radio-button
+                        uxTabbableListItem
+                        *ngFor="let option of filteredOptionList"
+                        name="group"
+                        [value]="selected"
+                        (valueChange)="selected = $event"
+                        [option]="option"
+                        (keydown.space)="selectButton($event, option)">
+                        <span [innerHTML]="option.name | highlightSearch: filter">
+                        </span>
+                    </ux-radio-button>
+                </div>
+            </ng-template>
+
+        </ux-select-custom>
+    </div>
+</div>
+
+<div class="row uxd-customize-example">
```

Comment:
Could you remove this section from the code snippet? It adds a lot of code to scroll through that is not relevant to the example. Also in plunker you can change the property values directly.

Reply:
Done.